nifi-registry/nifi-registry-core/nifi-registry-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -279,6 +279,22 @@ The default UserGroupProvider is the FileUserGroupProvider, however, you can dev
 
 NOTE: Initial User Identities are only created if the specified Users File is missing or empty during NiFi Registry startup. Changes to the configured Initial Users Identities will not take effect if the Users File is populated.
 
+===== DatabaseUserGroupProvider
+
+If you are using database for storing buckets and flows (like described in the 'Database Properties' section above), you can store the users, groups and policies in the same database either. 
```

Comment:
Minor nitpick, I suggest "as" instead of "like" and "as well" instead of "either".